journal/original/32-epistles.adoc

```diff
@@ -458,7 +461,8 @@ Your friend and brother,
 [.signed-section-context-close]
```

Comment:
This needs to be a `[signed-section-signature]` instead of a `[.signed-section-context-close]` 👍